apps/frontend/src/app/PageCharacter/CustomMultiTarget.tsx

```diff
@@ -110,7 +110,7 @@ function validateCustomTarget(ct: unknown): CustomTarget | undefined {
   if (
```

Comment:
Should we adjust the default (line 91) to be 'global'?

Reply:
I'd say we make it opt-in for now, we can always make it the default later, if there is an overwhelming request for it to be